internal/provider/exemption_resource.go

```diff
@@ -67,6 +67,12 @@ func (e *exemptionResource) Schema(_ context.Context, _ resource.SchemaRequest,
 				Default:     defaultEmptyList{},
 				Description: "The array of labels to keep; labels not in this array will be aggregated.",
 			},
+			"disable_recommendations": schema.BoolAttribute{
+				Optional:    true,
+				Computed:    true,
+				Default:     defaultBoolFalse{},
+				Description: "When set to true, the recommendations service will exempt this metric from consideration. When set to false, the recommendations service will consider this metric to be exempt from aggregation.",
```

Comment:
> When set to false, the recommendations service will consider this metric to be exempt from aggregation

I think we should remove this piece, since keep_labels must also be empty for that to be true

Reply:
Good point - done!